plugin.go

```diff
@@ -2,31 +2,32 @@
 package plasmactlbump
 
 import (
-	"github.com/spf13/cobra"
-
 	"github.com/launchrctl/keyring"
 	"github.com/launchrctl/launchr"
+	"github.com/spf13/cobra"
+
```

Comment:
- [ ] collecting XXX dependencies: > can you please uppercase the C ?

Reply:
changed